spec.html

```diff
@@ -172,6 +174,9 @@ <h1>PerformRealmEval ( _sourceText_, _callerRealm_, _evalRealm_ )</h1>
 			<emu-note type=editor>
 				Some steps from PerformRealmEval are shared with |eval| and |Function| and should result into a shared abstraction when merged to ECMA-262.
 			</emu-note>
+			<emu-note>
+				This abstraction requires the performed evaluation to result into a normal completion. Otherwise, if the result is not a normal completion, the abstraction will throw a TypeError exception associated to its original running execution context.
+			</emu-note>>
```

Comment:
In case it hasn't already been cleaned up, there's a typo here.
```suggestion
			</emu-note>
```

Reply:
thanks! I got that earlier today and fixed it already.